sklearn/plot/_confusion_matrix.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_confusion_matrix(values, classes, normalize=False,
```

Comment:
I wouldn't return anything from a plotting function, and if we provide ``y_true`` and ``y_pred`` we should also have an optional ``sample_weights``.